examples/roundtrip/ping.cpp

```diff
@@ -169,10 +171,14 @@ static bool parse_args(int argc, char *argv[])
 {
   if (argc >= 2)
   {
-    if (0 == strcmp(argv[1], "-l"))
+    if (0 == strcmp(argv[1], "-l")) {
       use_listener = true;
-    else if (0 == strcmp(argv[1], "-h"))
+    } else if (0 == strcmp(argv[1], "-h")) {
```

Comment:
I guess the -h option is supposed to print help, right? Then why does specifying this option result in the application exiting with EXIT_FAILURE? I can imagine it does that when specifying the wrong parameters, but asking for help does not seem a reason to report a FAILURE to me....